app/cdap/components/ColumnInsights/Components/ColumnDetails/index.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { DATATYPE_OPTIONS } from 'components/ColumnInsights/options';
+import React, { useState } from 'react';
+import { useStyles } from 'components/ColumnInsights/Components/ColumnDetails/styles';
+import EditIcon from '@material-ui/icons/Edit';
+import { Box, Typography } from '@material-ui/core';
+import InputSelect from 'components/ColumnInsights/Components/InputSelect';
+import { IColumnDetailsProps } from 'components/ColumnInsights/Components/ColumnDetails/types';
+import T from 'i18n-react';
+
+const PREFIX = 'features.NewWranglerUI.ColumnInsights';
```

Comment:
Please move this to constants file. Also please make similar change in other files if it is repeated.

Reply:
done